Extensions/BBText/bbtextruntimeobject.js

```diff
@@ -1,8 +1,26 @@
+//TODO @blurrymind correct the descriptions.
```

Comment:
@blurymind I am not sure this is your extension right maybe you could check out if I did that correctly

Reply:
I think this was corrected by @4ian now